README.md

```diff
@@ -1,16 +1,204 @@
-# UCAN Invocation
+# UCAN Invocation Specification v0.1.0
 
 ## Editors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
 
 ## Authors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
+* [Irakli Gozalishvili](https://github.com/Gozala), [DAG House](https://dag.house/)
 
-# 0. Abstract
+## Depends On
+
+* [`ucan-ipld`](https://github.com/ucan-wg/ucan-ipld/)
+
+# 0 Abstract
 
 ## Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
+# 1 Introduction
+
+## Separation
+
+Why separate the UCAN from the invocation format? Surely the UCAN itself already contains all the infomation required.
+
+I would argue that this is not the case for a few reasons.
+
+1. Authority is not intent to execute. Authority transfer is first-class in UCAN, other actions are not
+2. Mixing the two levels means that invocation info will live with the token (as a proof) on subdelegations
+3. Other detail, such as scheduling
+
+A job request may be picked up by anyone, PRIOR to UCAN delegation. A handshake or matchmaking may need to be performed.
```

Comment:
This does resonates. Yet lately we've been thinking about `aud` more as program we'd like to run and less like a actor that needs to run it.

Reply:
haha oops that got copy/pasted from IPVM 😅

We've been modelling "the program to run" as the resource.

```js
{
  with: "myprogram",
  do: "execute",
  nb: [{args: [1,2,3]}]
}
```

Can you give me a concrete example of putting a program in the `aud` field?

Reply:
To expand on this a bit: the distinction that's becoming clear in the IPVM context is that we want to make a deal with "someone". Once we get an agreement with a runner, we transmit the actual authority to run those actions, any secrets that they need (e.g. AES keys), and so on.

In IPVM, only _after_ they've agreed to run it do they get a signed invocation.